table.csv

```diff
@@ -496,3 +502,7 @@ skynet-ns,                      namespace,      0xb19910,       draft,     Skyne
 arweave-ns,                     namespace,      0xb29910,       draft,     Arweave Namespace
 subspace-ns,                    namespace,      0xb39910,       draft,     Subspace Network Namespace
 kumandra-ns,                    namespace,      0xb49910,       draft,     Kumandra Network Namespace
+es256,                          varsig,         0xd01200,       draft,     ES256 Signature Algorithm
```

Comment:
why are these all the way up here?

Reply:
There's a few places on this table that are like this, following the same pattern as we have here. The table enforces sorting by the code as if it were a number, not as sequence of bytes. `0xd999` is sorted as "lower" than `0xd0000`.

Reply:
These ended up separated because their public keys are also a byte longer than the other public key group 🤷‍♀️

Reply:
FWIW, I would love to get this table sorted as a byte array / lexicographically rather than numerically

Reply:
> sorting by the code as if it were a number, not as sequence of bytes

Since the main consumption of values in this table is as integers encoded as varints, I think this is logical; I can see how you might want it otherwise but I don't think we have many uses that view these as bytes rather than numbers.

When someone eventually turns this into a .json, with the .csv a "compiled" form of it, we could potentially also compile other forms for different kinds of views of the same data where that's useful to folks.